pulsar-client/src/main/java/org/apache/pulsar/client/impl/BinaryProtoLookupService.java

```diff
@@ -58,9 +60,11 @@ public class BinaryProtoLookupService implements LookupService {
     private final PulsarClientImpl client;
     private final ServiceNameResolver serviceNameResolver;
     private final boolean useTls;
-    private final ExecutorService executor;
+    private final ExecutorService scheduleExecutor;
```

Comment:
I guess this should be called "lookupExecutor" to be consistent?

Reply:
Just rename it to `scheduleExecutor` from `executor`.

Maybe this can use the `lookupPinnedExecutor`, I can make a new PR to improve this.